internal/clients/nightfall/nightfall.go

```diff
@@ -211,6 +217,59 @@ func (n *Client) createScanRequest(items []string) nightfallAPI.ScanRequest {
 	}
 }
 
+func (n *Client) scanContent(ctx context.Context, cts []*contentToScan, requestNum int, logger logger.Logger) ([]*diffreviewer.Comment, error) {
+	// Pull out content strings for request
+	items := make([]string, len(cts))
+	for i, item := range cts {
+		items[i] = item.Content
+	}
+
+	// send API request
+	resp, err := n.Scan(ctx, logger, items)
+	if err != nil {
+		logger.Debug(fmt.Sprintf("Error sending request number %d with %d items", requestNum, len(items)))
+		return nil, err
+	}
+
+	// Determine findings from response and create comments
+	createdComments := createCommentsFromScanResp(cts, resp, n.DetectorConfigs, n.TokenExclusionSet)
+	logger.Debug(fmt.Sprintf("Got %d annotations for request #%d", len(createdComments), requestNum))
+	return createdComments, nil
+}
+
+func (n *Client) scanAllContent(ctx context.Context, logger logger.Logger, cts []*contentToScan, commentCh chan []*diffreviewer.Comment) {
```

Comment:
nit - i think we can be explicit about this channel being write only (`commentCh chan <- []*diffreviewer.Comment`)